src/backbone.syphon.typeregistry.js

```diff
@@ -33,7 +37,7 @@ _.extend(TypeRegistry.prototype, {
 
   // Remove an item from a given type registration
   unregister: function(type) {
-    if (this.registeredTypes[type]) {
+    if (_.has(this.registeredTypes,type)) {
```

Comment:
Please pop in a space for formatting:

`if (_.has(this.registeredTypes, type)) {`